dlang.org.ddoc

```diff
@@ -242,16 +242,16 @@ $(SUBMENU
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU
+    $(ROOT_DIR)acknowledgements.html, Acknowledgments,
+    https://wiki.dlang.org/Books, Books,
     $(ROOT_DIR)tools.html, D-Specific Tools,
```

Comment:
Probably a better Wiki page would be https://wiki.dlang.org/Development_tools .

Anyway, the other question is that the pages for rdmd and htod should still be reachable on dlang.org (without having to go through the wiki).